modal/app.py

```diff
@@ -240,6 +246,28 @@ async def deploy(self, name: str, namespace, object_entity: str) -> str:
         deploy_response = await retry_transient_errors(self._client.stub.AppDeploy, deploy_req)
         return deploy_response.url
 
+    async def spawn_sandbox(
```

Comment:
not super important since i'm refactoring this code anyway, but i think it's more consistent if you had a `Sandbox` class here which is a provider class, similar to all other objects. and then `spawn_sandbox` would mostly just call a static constructor factory on `Sandbox`. but we can fix that later once we've merged providers and handles